lab2.js

```diff
@@ -98,21 +133,45 @@ function sayHello (sb) {
     // of the listener (the sb parameter above).
     // use the 'hello' object at the beginning of this exercise
     // to do the translating
-
-    //TODO: put this on the SentientBeing prototype
+    console.log(hello[this.language]);
+    return hello[sb.language];
   }
+  //TODO: put this on the SentientBeing prototype
+SentientBeing.prototype = {};
+SentientBeing.prototype.sayHello = sayHello;
+
+function Klingon() {
+  this.parent = 'SentientBeing';
+}
+function Romulan() {
+  this.parent = 'SentientBeing';
+}
+function Human() {
+  this.parent = 'SentientBeing';
+}
+
 
 // TODO: create three SentientBeings, one for each language in the
 // 'hello' object above.
-var klingon = new SentientBeing(); // TODO: make a klingon
-var romulan = new SentientBeing(); // TODO: make a romulan
-var human = new SentientBeing(); // TODO: make a human
+var Klingon = new SentientBeing('Qo\'noS', 'nuqneH'); // TODO: make a klingon
```

Comment:
here, you want to pass in the language name (look at what's in the hello variable), not the word for hello. Remember, you're using that language name to look up 
hello[language]
in your translation function.